### PR TITLE
limit record title field before custom tpl are pulled in

### PR DESCRIPTION
**Pull request by jeremyrl**

### CODE REVIEW

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- LEAF_Request_Portal/js/titleValidator.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=4877893&bid=16165370)

File: LEAF_Request_Portal/js/titleValidator.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var prevTitle = "";
+var maxTitleLength = 256;
+$(document).ready(function(){
+    prevTitle = $('input[name="title"][type="text"]').val();
+});
+$(document).on('DOMNodeInserted', 'input[name="title"][type="text"]', function(e) {
+    prevTitle = $('input[name="title"][type="text"]').val();
+    console.log("prevTitle: "+prevTitle);
```
Comment (thomasnguyen95) [inline LEAF_Request_Portal/js/titleValidator.js]: We can remove the console.log

Comment (jeremyrl) [reply LEAF_Request_Portal/js/titleValidator.js]: fixed

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- LEAF_Request_Portal/js/titleValidator.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=4877893&bid=16165370)

**Pull request merged**